tools/devtool

```diff
@@ -540,6 +539,20 @@ umount_ramdisk() {
     sudo rmdir ${DEFAULT_RAMDISK_PATH} &>/dev/null
 }
 
+ensure_ci_artifacts() {
+    # Fetch all the artifacts so they are local
+    say "Fetching CI artifacts from S3"
+    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.5/$(uname -m)
```

Comment:
Mh, if we hardcode the version here, will we need to update it as part of the release process?

Reply:
hmm yeah I imagined that we would want to be very explicit when doing this. Another option is to get the version from Cargo.